### DISCUSSION TITLE
"Name must be unique." Errors for issue form templates of forked repository

### DISCUSSION THREAD

**Discussion by anatawa12** [community/community#4460] [Projects and Issues]
If there's an issue form template in a non-default branch with the same name but a different file in upstream, The template system says 

```
There is a problem with this template


```

and GitHub doesn't show the yaml file as an issue form file.

I found this bug at https://github.com/anatawa12/KaizPatchX/blob/form-style-issue-template/.github/ISSUE_TEMPLATE/bug-report-en.yml however it may be merged so it may going to get not caused.

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
